src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -103,10 +107,13 @@ public static Export CreateExport<T>(object exportInstance, string contractName
             for (int i = 0; i < requiredExports.Count(); i++)
             {
                 // Try importing when not all of the required exports are available -> exception
-                Exceptions.Expect<Exception>(
```

Comment:
Switching to use the `FluentAssertions` `Should().Throw` fixes one of errors in `CheckTypeCanBeImported`.